R/pkg/tests/fulltests/test_mllib_classification.R

```diff
@@ -130,7 +130,7 @@ test_that("spark.logit", {
   summary <- summary(model)
 
   # test summary coefficients return matrix type
-  expect_true(class(summary$coefficients) == "matrix")
+  expect_true(any(class(summary$coefficients) == "matrix"))
```

Comment:
> matrix objects now also inherit from class "array", so e.g., class(diag(1)) is c("matrix", "array"). This invalidates code incorrectly assuming that class(matrix_obj)) has length one.

https://cran.r-project.org/doc/manuals/r-devel/NEWS.html

Reply:
Thanks! This also reminds me that it'll be great to test on r-devel (which I guess is 4.0.3 right now)